src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int numberOfFigures = 6;
```

Comment:
Magic number '6' should be extracted into a constant to improve code readability and maintainability. For example, private static final int NUMBER_OF_FIGURES = 6;